go.mod

```diff
@@ -56,11 +56,11 @@ require (
 	github.com/xiang90/probing v0.0.0-20190116061207-43a291ad63a2 // indirect
 	go.uber.org/zap v1.11.0 // indirect
 	golang.org/x/sys v0.0.0-20191127021746-63cb32ae39b2
-	k8s.io/api v0.0.0
-	k8s.io/apimachinery v0.0.0
+	k8s.io/api v0.17.0
```

Comment:
For the sake of consistency, please use `v0.16.4` for the k8s.io/api and apimachinery for `release-4.3`. This corresponds to k8s 1.16.

Reply:
Okay, I've set those in the `replace` section.  The `require` section still gets rewritten to mention newer versions by the tools, which is a little confusing, but I think it's right.

Reply:
@nalind with the proper tag, I don't think we need the `replace` stanzas for these packages any more. Upstream release engineering did a lot of work so that this should be possible. Can you give it a shot?